src/CMakeLists.txt

```diff
@@ -128,6 +128,15 @@ if(FLB_AWS)
     )
 endif()
 
+if(FLB_AWS)
+  set(src
+    ${src}
+    "aws/flb_aws_credentials.c"
+    "aws/flb_aws_credentials_sts.c"
+    "aws/flb_aws_util.c"
+    )
+endif()
```

Comment:
Something went wrong here- there is already an if block a few lines above for `FLB_AWS`. Remove this one, it is not needed.

Reply:
removed.